### PR TITLE
chore: Update mainnet-canisters.json

### PR DESCRIPTION
**Pull request by aterga**
This PR updates the pinned versions of mainnet NNS and SNS canisters after the last release cycle.

Swap: https://dashboard.internetcomputer.org/proposal/133383

SNS Governance: https://dashboard.internetcomputer.org/proposal/133384

SNS-W: https://dashboard.internetcomputer.org/proposal/133387

NNS Root: https://dashboard.internetcomputer.org/proposal/133386

NNS Governance: https://dashboard.internetcomputer.org/proposal/133385

Cycles Minting Canister: https://dashboard.internetcomputer.org/proposal/133311

### CODE REVIEW

**Review by basvandijk — APPROVED** [review]

File: mainnet-canisters.json [hunk 1]
```diff
@@ -20,24 +20,24 @@
     "sha256": "e6072806ae22868ee09c07923d093b1b0b687dba540d22cfc1e1a5392bfcca46"
   },
   "cycles-minting": {
-    "rev": "77f48ae63af09b6538b1bf33d3accc3bc74d14f8",
-    "sha256": "3260e795bd3e446a189539ce89d44cb29f7d196b92cdd2e2c75571c062ef1e50"
+    "rev": "ca8847547d327ce8a3bd81d25a590e01da1a3af5",
+    "sha256": "57f2cbbdbc2d8943173acc6ebd55aca9b9c12e2140b38ff5f20aea7b101ea570"
   },
```
Comment (daniel-wong-dfinity-org) [inline mainnet-canisters.json:25]: PR description does not mention cycles-minting. I guess we forgot to update this after previous release? We really need a robot to generate these PRs automatically...

Comment (aterga) [reply mainnet-canisters.json:25]: Good catch. I've added a link to the CMC proposal

File: rs/ledger_suite/tests/sm-tests/src/lib.rs [hunk 2]
```diff
@@ -2644,15 +2644,15 @@ pub fn test_upgrade_serialization(
...
+                    // Current mainnet ICP and ICRC wasms (V1) can both deserialize from memory manager
                     match env.upgrade_canister(
                         ledger_id,
                         ledger_wasm_mainnet.clone(),
                         upgrade_args.clone(),
                     ) {
                         Ok(_) => {
                             if !downgrade_to_mainnet_should_succeed {
-                                panic!("Downgrade from memory manager directly to mainnet should fail (since mainnet is V0)!")
+                                panic!("Downgrade from memory manager directly to mainnet should fail (since mainnet is V1)!")
```
Comment (maciejdfinity) [inline rs/ledger_suite/tests/sm-tests/src/lib.rs:2655]: V1 can deserialize from mem mgr. Maybe the comment should say simply "Downgrade to mainnet should fail!".

**Review by maciejdfinity — APPROVED** [review]

File: rs/ledger_suite/tests/sm-tests/src/lib.rs (hunk 2 above)
Comment (mbjorkqvist) [reply rs/ledger_suite/tests/sm-tests/src/lib.rs]: Changed as suggested!

**Pull request merged**